### ISSUE TITLE
Migrate 'Highlight Playhead' to plugin

### ISSUE THREAD

**Issue by randomeizer** [fcpxhacks/fcpxhacks#241]
Relates to #136.

**Comment by latenitefilms** [comment]
@randomeizer - Is this something you're waiting on me on to review?

**Comment by randomeizer** [comment]
Yeah, it's done. It includes the single and multicam match frames

**Comment by latenitefilms** [comment]
Ah, right, so it's already merged into 'develop' - it just hasn't been properly tested, which is why this issue is still open?

**Comment by randomeizer** [comment]
I don't think I merged it.

**Comment by latenitefilms** [comment]
No worries. I'll try and test it out tomorrow, and merge once tested.

Note to self - fix this:

 ```
> ERROR:   LuaSkin: hs.hotkey callback error: ...hammerspoon/hs/fcpxhacks/plugins/timeline/matchframe.lua:282: attempt to call a nil value (field 'deleteHighlight')
stack traceback:
	...hammerspoon/hs/fcpxhacks/plugins/timeline/matchframe.lua:282: in field 'matchFrame'
	...hammerspoon/hs/fcpxhacks/plugins/timeline/matchframe.lua:377: in function <...hammerspoon/hs/fcpxhacks/plugins/timeline/matchframe.lua:377>
	(...tail calls...) [23:44:58]
```

**Comment by randomeizer** [comment]
Cool
